lib/mutant/ast/regexp/transformer/quantifier.rb

```diff
@@ -0,0 +1,112 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        # Transformer for regexp quantifiers
+        class Quantifier < self
+          # Mapper from `Regexp::Expression` to `Parser::AST::Node`
+          class ExpressionToAST < Transformer::ExpressionToAST
+            # Transform quantifier into node
+            #
+            # @return [Parser::AST::Node]
+            def call
+              ast(expression.min, expression.max)
```

Comment:
Are these being called on a `Range`? If so, note that `#max` is implemented using `Enumberable#max` which could be expensive given a large range. Assuming the Range is inclusive you can just use `#begin` and `#end`. If it's an exclusive Range, assuming it's an `Integer`, you could use `range.end - 1` for the last element.